dev-docker-compose.yml

```
@@ -0,0 +1,104 @@
+    networks:
+      - app-network
+    healthcheck:
+      test: ["CMD", "redis-cli", "ping"]
+      interval: 10s
+      timeout: 5s
+      retries: 5
+
+  app:
+    build:
+      context: .
+      dockerfile: Devdockerfile 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)